public/js/data.js

```diff
@@ -441,8 +441,9 @@ function save() {
     if (!userSignedIn) {
         // user not signed in, save locally temporarily and force user to sign in
         localStorage.setItem("recover_login", data);
```

Comment:
Do we need to store this data in local storage when we user opt not to redirect ?

Reply:
I guess not. I felt in case if he clicks on save and his browser or machine crashes then it will be saved in its local storage right?